Android/app/src/main/java/app/intra/net/doh/google/GoogleServerConnection.java

```diff
@@ -45,7 +45,8 @@
 
   private static final String LOG_TAG = "GoogleServerConnection";
 
-  private static final String HOSTNAME = "dns.google.com";
+  static final String HOSTNAME = "dns.google.com";
```

Comment:
Let's rename to `HTTP_HOSTNAME` so it's not ambiguous.

Reply:
Done in 15a592c.